loadflow/loadflow-validation/src/main/java/com/powsybl/loadflow/validation/CandidateComputations.java

```diff
@@ -41,7 +40,7 @@ public static List<CandidateComputation> getComputations() {
      * Get the list of all known candidate computations names.
      */
     public static List<String> getComputationsNames() {
-        return getComputations().stream().map(CandidateComputation::getName).collect(Collectors.toList());
+        return getComputations().stream().map(CandidateComputation::getName).toList();
```

Comment:
Could you precise in the javadoc that the returned list is immutable… and add this same information in the `getComputations`'s one?

I think that returning an immutable list here is legitimate, but this change of behavior should be clearly stated. I will add a notice in the PR description.